problems/3.conditionals.js

```diff
@@ -3,24 +3,38 @@
 * @returns {string} - A message about the temperature
 */
 function temperatureMessage() {
-
+  function temperatureMessage(temp) {
```

Comment:
This logic is perfect, you just shouldn't have lines 6 - 12 nested inside the function you're supposed to write. The inner function has all the logic and should be the only thing here.